.github/workflows/build.yml

```diff
@@ -0,0 +1,326 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  schedule:
+    - cron: "0 7 * * *"
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  # Latest version is at https://github.com/Azure/azure-cli/releases
+  azCliVersion: 2.6.0
```

Comment:
Revisit this after first pipeline execution to make it  2.23.0.

Reply:
Will create separate issue for updating